grid/address/ip-geolocation/maps/ipdata.suma

```diff
@@ -0,0 +1,70 @@
+profile = "address/ip-geolocation@1.0"
+provider = "ipdata"
+
+// IPData documentation: https://docs.ipdata.co/
+map IpGeolocation {
+  ipAddress = ''
+
+  set if (input.ipAddress) {
+    ipAddress = input.ipAddress
+  }
+
+  http GET "/{ipAddress}" {
+    security "apikey"
+    response 200 "application/json" {
+      timezone = undefined
+
+      set if (body.time_zone) {
+        timezone = body.time_zone.name
+      }
+
+      map result {
+        ipAddress = body.ip
+        addressCountryCode = body.country_code ? body.country_code : undefined
+        addressCountry = body.country_name ? body.country_name : undefined
+        addressRegion = body.region ? body.region : undefined
+        addressLocality = body.city ? body.city: undefined
+        postalCode = body.postal ? body.postal : undefined
+        timeZone = timezone
+        latitude = body.latitude ? body.latitude : undefined
+        longitude = body.longitude ? body.longitude : undefined
```

Comment:
Do they return `null` or an empty string that you need to explicitly provide `undefined` here?

Reply:
Yes they return `null` values.

```
{
   "ip":"8.8.8.8",
   "is_eu":false,
   "city":null,
   "region":null,
   "region_code":null,
   ...
```